lib/state/stateManager.js

```diff
@@ -11,557 +11,524 @@ const rlp = utils.rlp
 
 /**
  * Interface for getting and setting data from an underlying
- * state trie
- * @interface StateManager
```

Comment:
Not have a strong opinion about this, just for clarification: did this not fit enough the [definition of the tag](http://usejsdoc.org/tags-interface.html)?